### PR TITLE
Implement fix from Edward Hague to fix Object type list length. Test …

### PR DESCRIPTION
**Pull request by GauiStori**
I have tested this patch and it fixes the BIT00008 in BTS

### CODE REVIEW

File: src/bacnet/bacdef.h [hunk 1]
```diff
@@ -109,7 +109,7 @@
 #elif (BACNET_PROTOCOL_REVISION == 24)
 /* Addendum 135-2020ca, 135-2020cc, 135-2020bv */
 #define MAX_ASHRAE_OBJECT_TYPE 65
-#define MAX_BACNET_SERVICES_SUPPORTED 47
+#define MAX_BACNET_SERVICES_SUPPORTED 49                /* 2024-05-15 mods4bts This was duplicated elsewhere. BACNET_SERVICES_SUPPORTED_MAX (bacenum.h) cr3817423048721093412*/
```
Comment (skarg) [inline src/bacnet/bacdef.h:112]: The extra comment should reside in a pull request discussion, but not in the final code.  Please remove the comment.

File: src/bacnet/bacenum.h [hunk 2]
```diff
@@ -1619,11 +1619,14 @@ typedef enum BACnet_Services_Supported {
     SERVICE_SUPPORTED_UTC_TIME_SYNCHRONIZATION = 36,
     SERVICE_SUPPORTED_WHO_HAS = 33,
     SERVICE_SUPPORTED_WHO_IS = 34,
-    SERVICE_SUPPORTED_UNCONFIRMED_AUDIT_NOTIFICATION = 46,
+    SERVICE_SUPPORTED_UNCONFIRMED_AUDIT_NOTIFICATION = 46
     /* Other services to be added as they are defined. */
     /* All values in this production are reserved */
     /* for definition by ASHRAE. */
-    BACNET_SERVICES_SUPPORTED_MAX = 47
+
```
Comment (skarg) [inline src/bacnet/bacenum.h:1626]: The extra comment should reside in a pull request discussion, but not in the final code.  Please remove the comment (and the line of code that is being removed).

File: src/bacnet/bacenum.h [hunk 3]
```diff
@@ -1598,8 +1598,8 @@ typedef enum BACnet_Services_Supported {
     SERVICE_SUPPORTED_PRIVATE_TRANSFER = 18,
     SERVICE_SUPPORTED_TEXT_MESSAGE = 19,
     SERVICE_SUPPORTED_REINITIALIZE_DEVICE = 20,
-    SERVICE_SUPPORTED_WHO_AM_I = 47,
-    SERVICE_SUPPORTED_YOU_ARE = 48,
+    SERVICE_SUPPORTED_WHO_AM_I = 47,    /* 2016 + bz    22*/
```
Comment (skarg) [inline src/bacnet/bacenum.h:1601]: Please use the same form that BACnetObjectType enumeration is using for the addendum comment:
```
/* Addendum 135-2016bz */
```

**Review by skarg — CHANGES_REQUESTED** [review]

File: src/bacnet/bacenum.h (hunk 3 above)
Comment (GauiStori) [reply src/bacnet/bacenum.h]: Thanks Steve. The comments are fixed in my latest commit.

**Pull request merged**